src/models/test/test.ts

```
@@ -0,0 +1,60 @@
+      if (name !== 'constructor') {
+        Object.defineProperty(
+          derivedCtor.prototype,
+          name,
+          Object.getOwnPropertyDescriptor(baseCtor.prototype, name) || Object.create(null)
+        );
+      }
+    });
+  });
+}
+
+class ConcertTestModel extends ConcertTestBase {
```

Biome (2.1.2)

[error] This class is unsafely merged with an interface.

The interface is declared here.

The TypeScript compiler doesn't check whether properties defined in the interface are initialized in the class.

(lint/suspicious/noUnsafeDeclarationMerging)